### PR TITLE
fix(website): ensure the nav menu renders nicely on mobile systems

### PR DESCRIPTION
**Pull request by bdjnk**
Some mobile systems evidently don't have fonts which include [⤣](http://www.fileformat.info/info/unicode/char/2923/index.htm).

Additionally, evidently there are bugs ([for example](https://stackoverflow.com/questions/38781089/font-family-monospace-monospace#38781315)) on some systems relating to monospace font rendering regarding line height and other typographical niggles.

This pull request addresses these problems.

### CODE REVIEW

File: packages/website/builder/css/nav.css [hunk 1]
```diff
@@ -165,7 +163,7 @@
 
 .nav_toggle,
 .nav_toggle-label::before {
-  font-family: "Courier New", monospace;
+  font-family: monospace, monospace;
```
Comment (henri-hulski) [inline packages/website/builder/css/nav.css]: I think one `monospace` should be enough.

File: packages/website/builder/css/nav.css [hunk 2]
```diff
@@ -210,13 +208,22 @@
   color: #cacaca;
   position: absolute;
   right: 0;
-  font-family: "Courier New", monospace;
+  font-family: monospace, monospace;
```
Comment (henri-hulski) [inline packages/website/builder/css/nav.css]: Same here.

File: packages/website/builder/css/main.css [hunk 3]
```diff
@@ -101,3 +101,20 @@ pre > code {
   font-size: 14px;
   line-height: 24px;
 }
+
+@font-face {
+    font-family: 'cerebral icons';
+    src: url('/images/cerebral-icons.woff') format('woff');
```
Comment (henri-hulski) [inline packages/website/builder/css/main.css]: Is woff format enough for all supported browsers?

File: packages/website/builder/css/nav.css (hunk 1 above)
Comment (bdjnk) [reply packages/website/builder/css/nav.css]: No, there's a bug and this is the hack fix. One monospace does the wrong thing in various browsers which misbehave.

File: packages/website/builder/css/nav.css (hunk 2 above)
Comment (bdjnk) [reply packages/website/builder/css/nav.css]: Same here ;)

File: packages/website/builder/css/main.css (hunk 3 above)
Comment (bdjnk) [reply packages/website/builder/css/main.css]: [Yes.](http://caniuse.com/#feat=woff)

File: packages/website/builder/css/nav.css (hunk 1 above)
Comment (henri-hulski) [reply packages/website/builder/css/nav.css]: I see.

**Pull request merged**